src/structures/interfaces/TextBasedChannel.js

```diff
@@ -203,8 +203,9 @@ class TextBasedChannel {
    * Performs a search within the channel.
    * <warn>This is only available when using a user account.</warn>
    * @param {MessageSearchOptions} [options={}] Options to pass to the search
-   * @returns {Promise<Array<Message[]>>}
-   * An array containing arrays of messages. Each inner array is a search context cluster
+   * @returns {Promise<Object>}
```

Comment:
Gus added a typedef for this could you use that instead? I forgot what it's called :(